packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleTableColumnAnchorBaseTests.cs

```diff
@@ -0,0 +1,9 @@
+using Xunit;
+namespace NimbleBlazor.Tests.Unit.Components;
+
+/// <summary>
+/// Tests for NimbleTableColumn API on <see cref="NimbleTableColumnAnchor"/>
+/// </summary>
+public class NimbleTableColumnAnchorBaseTests : NimbleTableColumnTests<NimbleTableColumnAnchor>
```

Comment:
nit is C# like Java? Can't have multiple classes in one file?
Seems kinda unnecessary to make these separate files.

If its an expected convention / a lint rule that's fine.
Maybe we could put them under there own namespace and in a subfolder?
`NimbleBlazor.Tests.Unit.Components.GroupableColumn`
`NimbleBlazor.Tests.Unit.Components.FractionalWidthColumn`

Reply:
I think it very well may be a lint rule, but if not, I agree with @rajsite...would be nice to reduce the amount of files for empty classes.

Reply:
Seems like there's no issue with putting all the classes in one file, so I've done that.